src/Form/form-autosuggest.mdx

```diff
@@ -19,23 +19,21 @@ Form auto-suggest enables users to manually select or type to find matching opti
 
 ```jsx live
 () => {
-    const [selected, setSelected] = useState({ title: '' });
+    const [selected, setSelected] = useState('');
 
     return (
         <Form.Autosuggest
             floatingLabel="Programming language"
             aria-label="form autosuggest"
             helpMessage="Select language"
             errorMessageText="Error, no selected value"
-            value={selected.title}
-            onSelected={
-                (value) => setSelected(prevState => ({ selected: { ...prevState.selected, title: value } }))
```

Comment:
This example wasn't actually working, something really weird going on with state management